### PR TITLE
[Tour of Beam] Learning content for "Triggers" module

### PR DESCRIPTION
**Pull request by sirenbyte**
#22527

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/tour-of-beam/learning-content/triggers/processing-trigger/description.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Processing time trigger is a trigger in Apache Beam that fires based on the current processing time of the pipeline. Unlike the event time trigger that fires based on the timestamps of the elements, the processing time trigger fires based on the actual time that the elements are processed by the pipeline.
+
+Processing time triggers are used in windowing operations to specify when the window should be closed and its elements should be emitted. The processing time trigger can be set to fire after a fixed interval, after a set of elements have been processed, or when a processing time timer fires.
+
+The following accumulation modes are available with processing time triggers:
+
+`Discarding`: any late data is discarded and only the data that arrives before the trigger fires is processed.
+
+`Accumulating`: late data is included and the trigger fires whenever the trigger conditions are met.
```
Comment (alxp1982) [inline learning/tour-of-beam/learning-content/triggers/processing-trigger/description.md:25]: Please add example description and challenge

**Review by alxp1982 — CHANGES_REQUESTED** [review]
Please address comments

**Comment by kerrydc** [comment]
LGTM, thanks

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**